### DISCUSSION TITLE
参考文献编译不出来怎么办？

### DISCUSSION THREAD

**Discussion by Martian9119** [tuna/thuthesis#868] [Q&A] [answered]
我把我的参考文献复制到ref/refs里面，但是编译出来的参考文献还是之前示例里给的参考文献而不是我的，然后我把和thuthesis—example同名的文件都删掉再重新编译，编译出来的文件没有bbl文件，报错也显示No file thuthesis-example.bbl.我该怎么解决？

**Comment by note286** [comment]
你需要`\cite`。

**Reply by Martian9119** [reply]
我都有加\cite啊

**Reply by note286** [reply]
仅根据你的描述，无法复现你的问题。

**Comment by zepinglee** [comment, marked as answer]
<img width="750" alt="Screenshot 2023-05-01 at 11 03 56" src="https://user-images.githubusercontent.com/12290822/235395841-3e023cc2-e644-43e8-938f-1f76d4931976.png">

按照文档的方法进行编译。

**Comment by shanzhi2002-cell** [comment]
我遇到了同样的问题
